src/tests/operate.test.js

```diff
@@ -0,0 +1,8 @@
+import operate from '../logic/operate';
+
+test('operate function performs the correct operation', () => {
+  const result = operate('2', '3', '+');
+  expect(result).toBe('5');
+});
+
+// Write more tests for the operate function based on your logic
```

Comment:
Nice work so far. However, as part of the project requirements, kindly make sure that all functionalities and scenarios of the operate.js modules are covered using jest. :wink: